src/AboutUs/View/page.twig

```diff
@@ -3,7 +3,40 @@
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>O nas</h1>
+		<h1 class="u-center-text">Krótki opis naszego zespołu T</h1><br><br><br>
```

Comment:
Why so many br tags?  If you want to add some space between h1 and p you should style it with css